lib/css/components/notices.less

```diff
@@ -1,252 +1,267 @@
 }
 
 // Banner
 // TODO deprecate .s-banner (by turning it into a modifier on .s-notice)
 // This would reduce the amount of CSS we ship to the client and simplify our codebase
 .s-banner {
-    .construct-notice-component();
-    --_no-x-offset: 0;
-    //      When we use .s-banner, we need to adjust the padding-top on
-    //      the body tag. This class correctly adjusts the body padding ONLY if
-    //      the notice is one line. If it wraps to multiple lines, more classes or
-    //      (ideally) JS will need to be used to determine the notice's height
-    //      at the time of render. The padding value is determined like so:
-    //
-    //      50px (top bar) + 44px (notice height) - 1px (bottom border)
-    //
-    //      The borders subtraction are necessary to neatly tuck everything together.
-    //  ----------------------------------------------------------------------------
-    &__body-pt {
-        padding-top: 93px;
-    }
-    //  If you want to put the banner above the topbar
-    &.is-pinned {
-        z-index: calc(var(--zi-navigation-fixed) + 1);
-    }
-    // If you want to hide and reveal the banner
-    &[aria-hidden="true"] {
-        --_no-x-offset: -50px; // TODO convert to su vars
-        visibility: hidden;
+    --_no-x-offset: 0; // [1]
+    .construct-notice-component(s-banner);
+
+    &[aria-hidden="true"] { // If you want to hide and reveal the banner
+        --_no-x-offset: calc(var(--su48) + var(--su2) * -1) // -50px
         opacity: 0;
+        visibility: hidden;
     }
     &[aria-hidden="false"] {
-        --_no-x-offset: 49px; // TODO convert to su vars
-        visibility: visible;
+        --_no-x-offset: calc(var(--su48) + var(--su1)); // 49px
         opacity: 1;
+        visibility: visible;
+    }
+    &.is-pinned { //  If you want to put the banner above the topbar
+        z-index: calc(var(--zi-navigation-fixed) + 1);
+    }
+    &__body-pt {
+        padding-top: 93px;
     }
-    // When we want to keep hero content capped
-    .s-banner--container {
+    & &--container { // When we want to keep hero content capped
+        margin: 0 auto;
+        max-width: calc(var(--s-step) * 10);
         position: relative;
         width: 100%;
-        max-width: calc(var(--s-step) * 10);
-        margin: 0 auto;
     }
 
+    -webkit-transform: translate3d(0, var(--_no-x-offset), 0);
+    transform: translate3d(0, var(--_no-x-offset), 0);
+
     border-width: var(--su-static1) 0;
     inset: 0 0 auto 0;
     padding: var(--su12);
     position: fixed;
     width: 100%;
     z-index: calc(var(--zi-navigation-fixed) - 1); // Tuck below topbar
-    -webkit-transform: translate3d(0, var(--_no-x-offset), 0);
-    transform: translate3d(0, var(--_no-x-offset), 0);
 }
 
 // Notice
 .s-notice {
-    .construct-notice-component();
+    .construct-notice-component(s-notice);
+
     border-radius: var(--br-sm);
     border-width: var(--su-static1);
     padding: var(--su16);
 }
 
 // Toast
 .s-toast {
```

Comment:
I'd love for `.s-toast` to instead be a modifier on `.s-notice`, but we should leave it this way for now.